region/brinstar/red.json

```diff
@@ -329,7 +333,11 @@
                   "canTrickyWalljump",
                   {"and": [
                     "SpaceJump",
-                    "canKillRippers"
+                    {"enemyKill":{
+                      "enemies": [
+                        [ "Ripper","Ripper","Ripper" ]
```

Comment:
I think there's 4 Rippers here too? They're just split across two nodes in (perhaps overzealous) anticipation for walljump things.

Reply:
Yeah that ripper at node 1 fooled me. Changing that as well.